src/main/resources/com/nirima/jenkins/plugins/docker/builder/DockerBuilderPublisher/help-labels.html

```diff
@@ -0,0 +1,4 @@
+<div>
```

Comment:
If we follow the getter/setter convention as used by the tags field, this file will have to be renamed to `help-labelsString.html`.
See earlier comments for full explanation.